inst/js/sidebar.js

```diff
@@ -1,12 +1,22 @@
 // used to collapse and expand the filter panel in teal apps
+/*
+resize is placed at end of functions
+b/c in embedded apps it will throw errors that cause the function to exit early
+*/
 var filter_open = true;
 const hideSidebar = () => {
-  $(".teal_secondary_col").fadeOut(1);
-  $(".teal_primary_col").attr("class", "teal_primary_col col-sm-12").resize();
+  $(".teal_secondary_col").css("display", "none");
+  $(".teal_primary_col").attr("class", "teal_primary_col col-sm-12");
```

Comment:
I think we can improve here a bit by only changing specific classes

```suggestion
  $(".teal_primary_col")
    .removeClass("col-sm-9")
    .addClass("col-sm-12");
```